### DISCUSSION TITLE
PC no longer boots after running

### DISCUSSION THREAD

**Discussion by toolsandutilities** [Rem0o/FanControl.Releases#472] [Q&A] [answered]
I ran FanControl and it scanned all my devices. After running it, my PC no longer boots properly. The power would turn on but the BIOS would not start and there is no video output. I have to fully reset the BIOS (clearing CMOS or re-flashing) for the BIOS to start and boot to Windows, which itself is taking 10 times longer than before.
I have done a lot of diagnostics and reinstalled a lot of drivers. The issue re-occurs when I fully shut down Windows and start it up again, forcing me to reset the BIOS. There is no issue when I restart. If I dual-boot to Linux, there is no issue when regardless if I shut down or restarted.
Does FanControl change some registry or BIOS setting? There seems to be some Windows setting caused by FanControl that bricks the PC.

**Comment by Rem0o** [comment, marked as answer]
Any change made by FanControl is volatile, it will get wiped out when the pc goes to sleep or you close/restart it. At every boot or wake from sleep, the config needs to be applied. With that in mind, I don’t see how the software could cause such issue.  Could be a bad ram stick, bad gpu, lacking psu…. Good luck with your diagnostic.
